resources/pgp-keys-map.list

```diff
@@ -867,22 +874,15 @@ org.hamcrest                    = \
                                   0x4DB1A49729B053CAF015CEE9A6ADFC93EF34893E, \
                                   0xE3A9F95079E84CE201F7CF60BEDE11EAF1164480
 
-org.hibernate:hibernate-core:5.6.3.Final \
-                                = noSig
-
-org.hibernate.common:hibernate-commons-annotations:5.1.2.Final \
-                                = noSig
-
-org.hibernate.validator:hibernate-validator:(,6.2.0.Final] \
-                                = noSig
+org.hibernate.*                 = noSig
```

Comment:
it's very interesting ... but true 
but hibernate says https://hibernate.org/community/keys/

Reply:
Good find. It's quite astonishing how many very heavily-used libraries are completely unsigned.

Even for the ones which are signed, it's nigh on impossible to verify the key used.